modules/dns/dns_api.rb

```
@@ -63,6 +63,8 @@ def dns_setup(opts)
       begin
         dns_setup({:fqdn => fqdn, :value => value, :type => type})
         @server.remove
+      rescue Proxy::Dns::NotFound => e
```

Nods, we should return the same error independent of the provider (client doesn't know which provider is being used, so they'll have to be defensive in treating our responses).

I'm not sure how much work it would be to add this disambiguation of possible errors to nsupdate providers, as we are parsing plain text there.